### ISSUE TITLE
[BS] Reloading Browsers browserfly stuck after changes 

### ISSUE THREAD

**Issue by NathanielMalca** [mgechev/angular2-seed#817]
Hey,

I have the last seed .

I'm using webstorm as editor in Windows.

I've copied all the files from my last seed to the app folder -> nmp start  = all is working just fine!

The problem is when i try to change code in ts or html template i get from browserfly in terminal:
 "[BS] Reloading Browsers..." and this is where is stays . refreshing chrome doesn't help.

in chrome console i see "Error: SyntaxError: Unexpected identifier(…) "Report this error at https://github.com/mgechev/angular2-seed/issues"

So basically the only way to make changes is to start the server again . its not possible to make changes on the fly .

Do you have any idea where to look in order to troubleshoot the problem? 

thanks

**Comment by mgechev** [comment]
Can't reproduce it but most likely you have a syntax error in your code.
